pkg/signing/signing.go

```diff
@@ -241,8 +241,8 @@ func getSigsForSelection(fimg *sif.FileImage, id uint32, isGroup bool) (sigs []*
 // will return true if the container is signed. Also returns a error
 // if one occures, eg. "the container is not signed", or "container is
 // signed by a unknown signer".
-func IsSigned(cpath, keyServerURI string, id uint32, isGroup bool, authToken string, noPrompt bool) (bool, error) {
-	noLocalKey, err := Verify(cpath, keyServerURI, id, isGroup, authToken, false, noPrompt)
+func IsSigned(cpath, keyServerURI string, id uint32, isGroup bool, authToken string, localVerify, noPrompt bool) (bool, error) {
+	noLocalKey, err := Verify(cpath, keyServerURI, id, isGroup, authToken, localVerify, noPrompt)
```

Comment:
I still don't understand what "local verify" means.

Reply:
LocalVerify is when your are verifying a container with your local keys (keys in your `~/.singularity/sypgp/pgp-public`), as opposed to keys in the keystore. During the `verify`, if you specify the `--local` flag, it will fail if the container is signed with a key thats not in your local keyring. So keys in your local keyring should only be keys that you trust.

_I hope all of that makes some sense..._